launcher/mainwindow_moc.cpp

```diff
@@ -68,45 +69,29 @@ MainWindow::MainWindow(QWidget * parent)
 		move(position);
 	}
 
-	//set default margins
+#ifdef Q_OS_IOS
```

Comment:
instead of checking for this flag, will it be possible to pass a parameter ENABLE_EDITOR from cmake?